index.js

```diff
@@ -115,7 +115,6 @@ LiftMasterAccessory.prototype = {
                 var thisAttributeSet = device.Attributes[j];
                 if (thisAttributeSet.AttributeDisplayName == "desc") {
                   thisDoorName = thisAttributeSet.Value;
-                  break;
```

Comment:
This break seems to assume that the "doorstate" attribute will have already been iterated through. In my case "doorstate" is later in the list and therefore breaking at this point means "doorstate" never gets set. Hence the removal of the break.